plugins/connection/incus.py

```diff
@@ -107,36 +117,40 @@ def _connect(self):
             )
             self._connected = True
 
-    def _build_command(self, cmd) -> list[str]:
+    def _build_command(self, cmd: str) -> list[str]:
         """build the command to execute on the incus host"""
 
         exec_cmd: list[str] = [
             self._incus_cmd,
             "--project",
             self.get_option("project"),
             "exec",
+            *(["-T"] if getattr(self._shell, "_IS_WINDOWS", False) else []),
             f"{self.get_option('remote')}:{self._instance()}",
             "--",
         ]
 
-        if self.get_option("remote_user") != "root":
-            self._display.vvv(
-                f"INFO: Running as non-root user: {self.get_option('remote_user')}, \
-                trying to run 'incus exec' with become method: {self.get_option('incus_become_method')}",
-                host=self._instance(),
-            )
-            exec_cmd.extend([self.get_option("incus_become_method"), self.get_option("remote_user"), "-c"])
+        if getattr(self._shell, "_IS_WINDOWS", False):
+            exec_cmd.extend(cmd.split(" "))
```

Comment:
I'm not familiar with the Windows syntax here but if it were Linux, I'd want to use `shlex.Split` instead to have something that's aware of argument quoting rather than just splitting on spaces.

Reply:
`shlex.split` is definitely not the right thing to call here, see my comment below.